test/sequential/test-repl-persistent-history.js

```diff
@@ -94,6 +100,17 @@ const tests = [{
   expected: [prompt, replDisabled, prompt]
 },
 {
+  env: { NODE_REPL_HISTORY_FILE: emptyHistoryPath },
```

Comment:
In that case, this test would test the same path as the test on line 108 since `oldHistoryPath === historyPath` as both would be `$HOME/.node_repl_history`.
This test exists to test the the `else if (oldHistoryPath)` branch.